app/editor/src/features/demo/DemoPage.tsx

```diff
@@ -132,7 +136,8 @@ export const DemoPage: React.FC = () => {
         // showFilter={true}
         // scrollSize="200px"
         // showHeader={false}
-        // groupBy={(item) => item.section}
+        // groupBy={(item) => item.original.section ?? 'any'}
```

Comment:
@jdtoombs It will now support both a predicate function or a property name string.